### PR TITLE
[New docs] docs' pagination

### PR DESCRIPTION
**Pull request by loic425**
| Q               | A
| --------------- | -----
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

[Preview](https://github.com/Sylius/SyliusResourceBundle/blob/8bdb41f96799c19a6832bdc1b2f50c24e70f82ba/docs/index.md)

### CODE REVIEW

File: docs/processors.md [hunk 1]
```diff
@@ -121,3 +121,5 @@ final class BoardGameResource implements ResourceInterface
 ```
 
 **[Go back to the documentation's index](index.md)**
+
+**[> Next chapter: Responders](responders.md)**
```
Comment (Roshyo) [inline docs/processors.md:125]: Where are responders.md?

Comment (loic425) [reply docs/processors.md:125]: https://github.com/Sylius/SyliusResourceBundle/blob/1e3e360f7b0fffb31dc8ff679159f45ceeecc88a/docs/responders.md

Comment (loic425) [reply docs/processors.md:125]: You could have a look on the preview.

File: docs/configure_your_operations.md [hunk 2]
```diff
@@ -386,3 +386,7 @@ class Book implements ResourceInterface
 | app_admin_book_bulk_delete | DELETE          | /admin/books/bulk_delete |    
 | app_shop_book_index        | GET             | /books/                  |
 | app_shop_book_show         | GET             | /books/{id}              |
+
+**[Go back to the documentation's index](index.md)**
+
+**[> Next chapter: Resource factories](resource_factories.md)**
```
Comment (Zales0123) [inline docs/configure_your_operations.md]: I suppose it should be `redirect.md` (see [here](https://github.com/Sylius/SyliusResourceBundle/pull/682/files#diff-b4d68dc855d0f9476d3f2ee343853bd21bf82ea9960d0cf06661baa244439dd6R25))?

Comment (loic425) [reply docs/configure_your_operations.md]: You're right, good point! I've updated the pagination and updated the preview, that seems ok now.

**Comment by Zales0123** [comment]
Thank you, Loïc! :tada:

**Pull request merged**